src/logger/loggerTaskEx.c

```diff
@@ -246,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
If you notice, the loggingSampleRate is a function of determining the highest sample rate present in the config.

Alternatively, the telemetrySampleRate is fixed regardless of the config. IIRC there's a case where it's possible that a certain configuration can prevent telemetry samples from being sent at all. That's why that extra code exists, to ensure that we at least send telemetry on those tick intervals.